src/agent/solana/exporter.rs

```diff
@@ -348,7 +374,7 @@ impl Exporter {
                     // Note: This message is not an error. Some
                     // publishers have different permissions on
                     // primary/secondary networks
-                    info!(
+                    debug!(
```

Comment:
We've seen complaints about these messages being too frequent. I forgot to mention it in my description. Sorry!

Reply:
thanks for commenting on the PR with context. definitely helps me understand why this changed.